### ISSUE TITLE
Better text and label examples

### ISSUE THREAD

**Issue by EldRoos** [sebgroup/green#743]
Many if not all components should have better names on components and it's descriptions in storybook so it easier can be tested with screen-readers and such.

Instead of:
Button Button Button

Have:
First Second Third

Or even realistic examples like:
Search Register Sign-in
